libs/ledger-live-common/src/families/cardano/js-synchronisation.ts

```diff
@@ -295,7 +295,7 @@ const makeGetAccountShape =
     );
 
     const utxos = prepareUtxos(newTransactions, stableUtxos, accountCredentialsMap);
-    let accountBalance = utxos.reduce((total, u) => total.plus(u.amount), new BigNumber(0));
+    let utxosSum = utxos.reduce((total, u) => total.plus(u.amount), new BigNumber(0));
```

Comment:
```suggestion
    const utxosSum = utxos.reduce((total, u) => total.plus(u.amount), new BigNumber(0));
```

cf. https://github.com/LedgerHQ/ledger-live/actions/runs/6967233140/job/18958758900#step:6:299

Reply:
Done